x-pack/libbeat/processors/add_cloudfoundry_metadata/add_cloudfoundry_metadata.go

```diff
@@ -79,18 +74,28 @@ func (d *addCloudFoundryMetadata) Run(event *beat.Event) (*beat.Event, error) {
 				"name": app.Name,
 			},
 			"space": common.MapStr{
-				"id":   app.SpaceData.Meta.Guid,
-				"name": app.SpaceData.Entity.Name,
+				"id":   app.SpaceGuid,
+				"name": app.SpaceName,
 			},
 			"org": common.MapStr{
-				"id":   app.SpaceData.Entity.OrgData.Meta.Guid,
-				"name": app.SpaceData.Entity.OrgData.Entity.Name,
+				"id":   app.OrgGuid,
+				"name": app.OrgName,
 			},
 		},
 	})
 	return event, nil
 }
 
+// String returns this processor name.
 func (d *addCloudFoundryMetadata) String() string {
 	return processorName
 }
+
+// Close closes the underlying client and releases its resources.
+func (d *addCloudFoundryMetadata) Close() error {
```

Comment:
This won't' be called right?

Reply:
Not at the moment, I included it to don't forget what to close if we finish https://github.com/elastic/beats/pull/16349

Reply:
I am also using it on tests.